components/google-cloud/google_cloud_pipeline_components/experimental/hyperparameter_tuning_job/component.py

```diff
@@ -0,0 +1,277 @@
+import json
+
+from kfp.components import create_component_from_func_v2
+from typing import NamedTuple
+
+def hyperparameter_tuning_job_run_op(
```

Comment:
can you go through the parameters and align the naming with vertex ai doc?
Also try as much as possible refer to the vertex ai doc for descriptions of the parameters.
We should do this for all new components. Building a wrapper with renames without good reason is not going to be a maintainable path going forward.

Reply:
Done.